eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Epoch.hs

```diff
@@ -256,26 +256,47 @@ epochTransition = do
         , reCurrentEpoch = eNo
         }
     govStateToSeq = Seq.fromList . Map.toList
```

Comment:
Going via a `List` could be avoided by folding over.

Reply:
This is going to be changed soon anyways. We'll have to use sequences for governance states so we won't have to convert from a map.